.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6-nightly'
+          - '^1.7.0-0'
```

Comment:
Is there a reason why this needs to be changed (and if so, perhaps we should put the docs in a different workflow?)

Reply:
Oh, sorry - no - silly me, I was thinking about Diffractor.

Reply:
I'll take it out.

Reply:
Changed to test on Julia v1 (since v1.6 is out now) and nightly - Ok?